### PR TITLE
Reduce actions

### PR DESCRIPTION
**Pull request by Daphineatim**
### In this milestone, the following were implemented;

- Added Redux Toolkit by installing react-redux @reduxjs/toolkit.
- Structured the application files using a "feature folder"  named redux.
-  Configured the Redux Store.
-  Wrote categories actions and reducer (in /src/redux/categories/categories.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
-  Wrote book's actions and reducer (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//     case REMOVE_BOOK:
+//       return {
+//         ...state,
+//         books: state.books.filter((book) => book.id !== action.payload),
+//       };
+//     default:
+//       return state;
+//   }
+// };
+
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] Kindly remove commented code

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export const checkStatusAction = () => ({
+  type: CHECK_STATUS,
+  payload: 'UNDER-CONSTRUCTION',
+});
+
+const categoriesReducer = (state = initialState, action = {}) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return action.type;
```
Comment (TedLivist) [inline src/redux/categories/categories.js]: - [x] This should be `action.payload`. Kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Daphineatim,

Good job so far!
✔️ Good documentation
✔️ Definition of action types
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Daphineatim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- PR is professional :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- No Linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**